src/Interfaces.ts

```diff
@@ -15,10 +16,17 @@ export interface IResolverValidationOptions {
     requireResolversForAllFields?: boolean;
 }
 
+export interface IResolverOptions {
+    resolve?: GraphQLFieldResolveFn;
+    __resolveType?: IResolveTypeFn;
+    __isTypeOf?: IIsTypeOfFn;
```

Comment:
we used to allow __description and __deprecationReason as well, but that can be done with comments and directives now, so we don't need to add it to typescript, because it would only be for backwards compatibility.